config/import-control.xml

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0"?>
 <!DOCTYPE import-control PUBLIC
-    "-//Checkstyle//DTD ImportControl Configuration 1.4//EN"
-    "https://checkstyle.org/dtds/import_control_1_4.dtd">
+    "-//Checkstyle//DTD ImportControl Configuration 1.5//EN"
+    "https://checkstyle.org/dtds/import_control_1_5.dtd">
 
 <import-control pkg="com.puppycrawl.tools.checkstyle">
-  <disallow pkg="java.util.concurrent"/>
+  <forbid pkg="java.util.concurrent"/>
   <!-- java.io.FileInputStream and java.io.FileOutputStream are disallowed in favor of
          Files.newInputStream and Files.newOutputStream.
          See https://github.com/checkstyle/checkstyle/issues/5638 -->
-  <disallow class="java.io.FileInputStream"/>
-  <disallow class="java.io.FileOutputStream"/>
+  <forbid class="java.io.FileInputStream"/>
+  <forbid class="java.io.FileOutputStream"/>
 
   <!-- guava causes issues with other libraries since not everyone is on the same version -->
-  <disallow pkg="com\.google.*" regex="true"/>
+  <forbid pkg="com\.google.*" regex="true"/>
```

Comment:
```suggestion
  <forbid package="com\.google.*" regex="true"/>
```

Reply:
i would suggest to do this in dedicated PR as well. what you think?

Reply:
separate PR is preferable.

But as we change version of dtd, we should merge both PR in same release.